ci_environment/redis/recipes/default.rb

```diff
@@ -21,12 +21,12 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-apt_repository "rwky-redis" do
-  uri          "http://ppa.launchpad.net/rwky/redis/ubuntu"
+apt_repository "chris-lea-redis-server" do
+  uri          "http://ppa.launchpad.net/chris-lea/redis-server/ubuntu"
   distribution node['lsb']['codename']
   components   ["main"]
-  key          "5862E31D"
-  keyserver    "hkp://ha.pool.sks-keyservers.net"
+  key          "136221EE520DDFAF0A905689B9316A7BC7917B12"
+  keyserver    "hkp://keyserver.ubuntu.com"
```

Comment:
If the key we need is available via the HA pool, I'd like to stick with that keyserver ❤️